src/ceph-disk/ceph_disk/main.py

```diff
@@ -18,6 +18,8 @@
 # GNU Library Public License for more details.
```

Comment:
minor: the commit message lists some commit subjects that you squashed - you can delete those